seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/resources/jdbc_mysql_source_and_sink_parallel_upper_lower.conf

```diff
@@ -29,8 +29,8 @@ source {
     query = "select * from source"
     partition_column = "c_bigint_30"
     result_table_name = "jdbc"
-    partition_lower_bound = 2844674407371055160
-    partition_upper_bound = 2844674407371055259
+    partition_lower_bound = 2844674407371055000
+    partition_upper_bound = 2844674407371055099
```

Comment:
Why change this?

Reply:
```
        BigDecimal bigintValue = new BigDecimal("2844674407371055000");
        BigDecimal decimalValue = new BigDecimal("999999999999999999999999999899");
        for (int i = 0; i < 100; i++) {
            ....
            decimalValue.add(BigDecimal.valueOf(i)),
            ....
        }
```

The field preset data ranges from 2844674407371055000 to 2844674407371055099, so this configuration is necessary to integrate all data